### PR TITLE
B. de las Casas, Apologia Ch. 3

### PR DESCRIPTION
**Pull request by a1wilson**
Drafted ch. 3
- several substitutions/strikethroughs, so please check for formatting

### CODE REVIEW

**Comment by jeffreycwitt** [comment]
Notice that this PR has 3 files changes even though you're working just one chapter. 

This occurring is because when you are starting a new branch, you are diverging from the branch you made for the previous chapter. 

Instead you should always change your branch to "master" and then start a new branch that diverges from master

File: lca8s8-d1e78/cod-dlc9qq_lca8s8-d1e78.xml [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+  </teiHeader>
+  <text xml:lang="la">
+    <front>
+      <div xml:id="starts-on">
+        <pb ed="#P" n="19-r"/>
+      </div>
+    </front>
+    <body>
+      <div>
+		<head id="P">Caput 3 [Bnf 12926 Transcription]</head>
```
Comment (jeffreycwitt) [inline lca8s8-d1e78/cod-dlc9qq_lca8s8-d1e78.xml]:
```suggestion
		<head>Caput 3 [Bnf 12926 Transcription]</head>
```

**Review by jeffreycwitt — CHANGES_REQUESTED** [review]

**Comment by jeffreycwitt** [comment]
I wonder if you're using a "schema aware" editor. I think you said you have a preference for emacs. That's fine. You should work in whatever editor environment enables you to do your best work. But it is generally a benefit to work in a "schema aware" editor that can enforce not just xml rules, but the customized TEI rules as well. The vscode environment I set up has this "rule-checking" feature which can help alert you to validation errors before you submit pull requests. If you want to stick with emacs, we should look for an emacs plugin that can support and enforce RelaxNG XML schema declarations. (Without this kind of tooling I think the editing experience will be frustrating. I certainly make lots of small mistakes that I only am able to correct because of effective tooling. We can discuss pros and cons next time we meet. )

**Comment by a1wilson** [comment]
- Thanks for pointing out my branching errors. I will make sure to branch from "master" in the future.
- I have a validation check, but not schema-awareness. I've been using Vim, actually. There's a plugin for RNG awareness/validation that I'll  try going forward.
Thanks for the advice to the newbie.

**Pull request merged**